addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -76,7 +76,7 @@
           {{/if}}
         </div>
 
-        {{#if (not-eq f.model.__typename "StaticQuestion")}}
+        {{#if (and (or (eq f.model.isRequired "true") (eq f.model.isRequired "false")) (not-eq f.model.__typename "StaticQuestion"))}}
```

Comment:
I'd extract this into the `js` file.

Reply:
I agree in principle. But as all the other display logic is in the template, I did so as well.

Reply:
Yeah, I think the "soft limit" for this kind of logic should be 2-3 assertions. Would you mind extracting it?

Reply:
"Soft limit"?

Anyways. Is there a good way to observe a Changeset in a computed property?

Reply:
Looks like you found out :+1:

Reply:
Nope. This doesn't work.

Reply:
maybe @anehx can help?